src/web/jdv/CMakeLists.txt

```diff
@@ -2,12 +2,11 @@ set(jdv_outdir ${project_SHARE_DIR}/jaiabot/web/jdv/)
 set(jdv_client_outdir ${jdv_outdir}/client/dist/)
 
 file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/server DESTINATION ${jdv_outdir})
-file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/client/dist/favicon.png DESTINATION ${jdv_client_outdir})
-file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/client/dist/index.html DESTINATION ${jdv_client_outdir})
 
 add_custom_command(OUTPUT ${jdv_client_outdir}/bundle.js
   COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/client/build.sh
   ARGS ${jdv_client_outdir}
+  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/client
   COMMENT "Running jdv build.sh"
   DEPENDS npm_depdendencies
```

Comment:
Same comment as with JCC: the DEPENDS should list all source files for webpack

Reply:
Ah yes, I think one of the next cards involves making sure the DEPENDS all include everything they really DEPEND on.